### ISSUE TITLE
Evaluate ad robustness

### ISSUE THREAD

**Issue by MartinLNG** [MartinLNG/adversarial_training_mps#166]
1. Ad robustness during pretraining (at least for one study, maybe not for the minimal version?)
2. Ad robustness after pretraining
3. Ad robustness during GAN training (maybe consider using some integrated performance metric (single positive number) as early stopping criterion
4. Ad robustness during adversarial training
5. Ad robustness during GAN-training with augmented data

maybe make ad robustness evaluation an option in the existing pretraining, and (seperated) gan training scripts.
